GameServerLib/Logic/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -67,6 +67,12 @@ public class Champion : ObjAIBase
             }
             Spells[4] = new Spell(this, clientInfo.SummonerSkills[0], 4);
             Spells[5] = new Spell(this, clientInfo.SummonerSkills[1], 5);
+
+            for(short i = 0; i < 6; i++)
+            {
+                Spells[(byte)(i + 6)] = new Spell(this, "", (byte)(i + 6));
```

Comment:
There's exactly what you would want to use as an empty spell: `BaseSpell`.
Riot uses these for non-champion objects, and I believe going that way is better than using an empty string.